src/hooks/disputable-voting-logic.js

```diff
@@ -10,6 +11,50 @@ function useGetVotes() {
   return votes
 }
 
+export function useGetVote(voteId) {
+  const [loading, setLoading] = useState(false)
+  const [extendedProperties, setExtendedProperties] = useState(null)
+  const [disputableVoting] = useApp('disputable-voting')
+  const [vote] = useDisputableVoting(disputableVoting, (app) => {
```

Comment:
No it wouldn’t update, and it definitely should yes. I think allowing the inline function is nice (rather than asking users to cache the reference with `useCallback()`), and we should have a dependencies array as well to control its updates.

Thanks for catching this! :see_no_evil: